manage_user_page.php

```diff
@@ -170,7 +170,7 @@
 	} else {
 		$t_title = '';
 	}
-	$t_active = $t_prefix === $f_filter ? 'active' : '';
+	$t_active = (string)$t_prefix === $f_filter ? 'active' : '';
```

Comment:
> It is strange that when the array key and value were set earlier they were already casted to a string

Exactly my thoughts when looking at the problem

The surprising output of the following script is: `string(1) "A" string(1) "B" int(2) string(1) "3"`
At least it explains the behavior.

```php
<?php

$t_prefix_array['A'] = 'B';
$t_prefix_array[(string)2] = (string)3;

foreach ( $t_prefix_array as $t_prefix => $t_caption ) {
    var_dump( $t_prefix, $t_caption );
}
```

> Is there a reason we have '===' specially if we are going to cast anyway?

'===' has been introduced to fix http://www.mantisbt.org/bugs/view.php?id=21115 see https://github.com/mantisbt/mantisbt/commit/413e641fc6b30ad0491044add4e999d1a2909df2